packages/d3fc-series/src/webgl/point.js

```diff
@@ -0,0 +1,83 @@
+import { glDraw, circlePointShader } from '@d3fc/d3fc-webgl';
+import xyBase from '../xyBase';
+import { rebindAll, exclude } from '@d3fc/d3fc-rebind';
+import scaleMapper from '@d3fc/d3fc-webgl/src/scale/scaleMapper';
+
+export default () => {
+    let context = null;
+    const base = xyBase();
+    let size = 70;
+
+    let draw = glDraw();
+
+    const point = (data) => {
+        const filteredData = data.filter(base.defined());
+        const program = draw.program();
+
+        const shaderBuilder = circlePointShader();
+        program.vertexShader(shaderBuilder.vertex());
+        program.fragmentShader(shaderBuilder.fragment());
+
+        program.numElements(filteredData.length);
+
+        const xScale = scaleMapper(base.xScale());
+        const yScale = scaleMapper(base.yScale());
+
+        xScale.glScale.coordinate(0);
+        yScale.glScale.coordinate(1);
+
+        const accessor = getAccessors();
+
+        const points = new Float32Array(filteredData.length * 3);
+        let pi = 0;
```

Comment:
`pi` is a slightly strange name for this variable! I was expecting `let pi = 3.141...`